### ISSUE TITLE
OWNERS.md Missing

### ISSUE THREAD

**Issue by kaovilai** [konveyor/java-analyzer-bundle#82]
OWNERS.md Missing per https://github.com/konveyor/community/blob/main/MAINTAINERS.md guidelines

**Comment by shawn-hurley** [comment]
I agree with @kaovilai, that across most of the konveyor repos we should add this, I think it should be coming soon.
